Traceback (most recent call last):
  File "/home/alien/Code/stackleberg_game/MARL/train.py", line 906, in <module>
    agent, stats = train(args)
  File "/home/alien/Code/stackleberg_game/MARL/train.py", line 398, in train
    next_state, _, rl, rf1, rf2, done, info = env.step(leader_action, follower1_action, follower2_action)
  File "/home/alien/Code/stackleberg_game/MARL/env/battery_disassembly_env.py", line 468, in step
    rl, rf1, rf2 = self.reward(state, al, af1, af2)
  File "/home/alien/Code/stackleberg_game/MARL/env/battery_disassembly_env.py", line 824, in reward
    rows_before = np.count_nonzero(np.any(state > 0, axis=1))
  File "/home/alien/anaconda3/envs/skgame/lib/python3.10/site-packages/numpy/core/fromnumeric.py", line 2412, in any
    return _wrapreduction(a, np.logical_or, 'any', axis, None, out,
  File "/home/alien/anaconda3/envs/skgame/lib/python3.10/site-packages/numpy/core/fromnumeric.py", line 88, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
numpy.exceptions.AxisError: axis 1 is out of bounds for array of dimension 1
